backend-next/migrations/0000-initial.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE IF NOT EXISTS messages (
+  id uuid primary key default gen_random_uuid(),
+  created_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  username varchar(128) NOT NULL,
+  orig_msg text NOT NULL,
+  tl_msg text,
+  country varchar(2)
+);
+
+CREATE TABLE IF NOT EXISTS gallery (
```

Comment:
What happened to the blurhash? Granted, I guess that makes things more complex, and we might not even really care...

EDIT: Oh, I see you're using skeleton later. That works.

Reply:
blurhash didn't get ported over to the new db since we have skeleton yea